tasks/rulegen/src/main.rs

```diff
@@ -316,16 +337,14 @@ impl<'a> Visit<'a> for TestCase {
     }
 
     fn visit_tagged_template_expression(&mut self, expr: &TaggedTemplateExpression<'a>) {
-        if expr.tag.is_specific_id("dedent") || expr.tag.is_specific_id("outdent") {
-            return;
-        }
-
-        // If it is a raw string like String.raw`something`, then we import that as a Rust raw string literal
         self.code = if expr.tag.is_specific_member_access("String", "raw") {
             expr.quasi.quasis.first().map(|quasi| format!("r#\"{}\"#", quasi.value.raw))
+        } else if expr.tag.is_specific_id("dedent") || expr.tag.is_specific_id("outdent") {
+            expr.quasi.quasis.first().map(|quasi| dedent(&quasi.value.raw).to_string())
         } else {
             expr.quasi.quasi().map(|quasi| quasi.to_string())
         };
```

Comment:
We should probably use `format_tagged_template_expression` here, right?

Reply:
Yep! But it requires extra clone overhead, I don't know if it's worth it.
```rust
let allocator = Allocator::default();

self.code = format_tagged_template_expression(&oxc_allocator::Box::new_in(
      CloneIn::clone_in(expr, &allocator),
      &allocator,
));

self.config = None;
```